src/main/kotlin/no/nav/amt/deltaker/bff/application/metrics/MetricRegister.kt

```diff
@@ -0,0 +1,40 @@
+package no.nav.amt.deltaker.bff.application.metrics
+
+import io.prometheus.client.CollectorRegistry
+import io.prometheus.client.Counter
+
+const val METRICS_NS = "amtdeltakerbff"
+
+class MetricRegister(
+    collectorRegistry: CollectorRegistry,
+) {
+    private val opprettetKladd: Counter = Counter.build()
+        .namespace(METRICS_NS)
+        .name("opprettet_kladd_count")
+        .help("Antall opprettede kladder for påmelding")
+        .register(collectorRegistry)
```

Comment:
Defaulten til `.register()` er allerede `CollectorRegistry.defaultRegistry` (noe som er uansett er statisk), så man har kunnet spart seg for å lage noe `collectorRegistry` i det hele tatt og gjort om denne klassen til et `object` eller rene funksjoner i steden, så man slipper å sende `MetricRegister` rundt om i alle servicene.

Reply:
ÅÅÅ! Nice! Da endrer jeg :D